integration/docker/info_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	. "github.com/kata-containers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("info", func() {
+	var (
+		stdout   string
+		exitCode int
+	)
+
+	Context("docker info", func() {
+		It("should has a runtime information", func() {
+			stdout, _, exitCode = dockerInfo()
+			Expect(exitCode).To(Equal(0))
+			Expect(stdout).To(ContainSubstring("Default Runtime: " + Runtime))
```

Comment:
I think we should allow the runtime being tested as not default runtime. This helps for people that has configured runc as default runtime.
For the rest of test we already (or should) speficy the runtime to be used to run containers.